logging.adoc

```diff
@@ -0,0 +1,397 @@
+
+
+*OTLP* describes the *protocol* for encoding, transporting, and delivering telemetry data between sources using the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry OTLP Specification]
+
+.Technical Preview
+This feature is currently in tech-preview and an annotation is required
+
+`observability.openshift.io/tech-preview-otlp-output: "enabled"`
+
+---
+== Configuring the Forwarder
+A new output type `otlp` has been defined in the `ClusterLogForwarder`
+
+.ClusterLogForwarder
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-logforwarder
+  namespace: my-app-namespace
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled"
+spec:
+  outputs:
+    - type: otlp  <1>
+      name: otel-collector
+      tls:
+        insecureSkipVerify: true  <2>
+      otlp:
+        url: 'http://my-otel-receiver-service:4318/v1/logs'  <3>
+        tuning:
+          compression: none  <4>
+        authentication:
+          token:
+            from: serviceAccount  <5>
+  pipelines:
+   - name: my-pipeline
+     inputRefs:
+     - application
+     - infrastructure
+     outputRefs:
+     - otel-collector
+  serviceAccount:
+    name: logger-admin
+----
+. Output `type` is '*otlp*'
+. `tls` includes the standard certificate configuration or specify `insecureSkipVerify`: '*true*'
+. `otlp` `url` specify a valid host:port of the otel receiver and *MUST* terminate with "*/v1/logs*"
+. `otlp` `tuning` `compression` default value is "*gzip*". Specify "*none*" to turn compression off.
+. `otlp` `authentication` is optional and specifies a `token` `from` and a value of "*serviceAccount*"
+.. The token can also be read from a secret
+.. Also available with `username` and `password` authentication spec (refer to HTTP Auth Specification for full scope)
+
+
+.TLS InsecureSkipVerify
+[NOTE]
+This option is *NOT* recommended for production configurations. If true, the client will be configured to skip validating server certificates.
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L244[TLS Specification] for _observability.openshift.io/v1_
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L203[HTTP Auth Specification] for _observability.openshift.io/v1_
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L812[Tuning Specification] for _observability.openshift.io/v1_
+
+
+== Data Model
+
+=== Semantic Convention
+The Semantic Conventions in OpenTelemetry define a *Resource* is an immutable representation of the entity producing telemetry as *Attributes*.
+
+For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
```

Comment:
Are these attribute names (e.g. container_name`)?

Reply:
See below.  These are created as resource attributes and reduced/grouped by their values.   `container_name` is mapped to: `k8s.container.name`